src/passes/CastRefining.cpp

```diff
@@ -61,7 +61,9 @@ struct NewFinder : public PostWalker<NewFinder> {
 };
 
 struct CastRefining : public Pass {
-  // Only changes cast types, but not locals.
+  // Changes types, either by refining them without changing nullability, or
+  // changing them to be null types, so we never add new non-nullable locals
+  // here.
```

Comment:
```suggestion
  // Changes types, either by refining them without changing nullability, or
  // changing them to be bottom types, so we never add new non-nullable locals
  // here.
```

Although "changing them to be bottom types" is already included in "refining them." The pass never changes nullability, even when refining to a bottom type.